src/commands/helm-deploy.yml

```diff
@@ -0,0 +1,98 @@
+description: >
+  Deploys Helm to ShuttleCrawler, assumes image version of component is set in .version
```

Comment:
Since this is a public orb, there are a few things that we should make generic and/or parameters:
- Shuttlecrawler references ("a kubernetes cluster" maybe?)
- `platform-` references (make a parameter)
- the failure slack message (parameter, it references a subteam that won't exist for other people plus we'll probably want to configure that per-project)

Reply:
Good catch, gonna make the changes if we keep it here, would it make more sense to move to a private orb though? Relevant slack convo: https://apollograph.slack.com/archives/C0240N1FGC9/p1668637598188009.